README.md

```diff
@@ -39,66 +39,81 @@
 
 
 
-<!-- ABOUT THE PROJECT -->
-## About The Project
+## About ApostropheCMS
 
-Our goal is to create a CMS for everyone in an organization. It's our belief that a CMS should inspire every one to express themselves on the web by providing a modern and flexible developer experience and an intuitive interface for content editors. ApostropheCMS is built for teams, and we want to make your job more fun.
+ApostropheCMS is content software for everyone in an organization. It helps teams of all sizes create dynamic digital experiences with elegance and efficiency by blending powerful features, developer happiness, and a low learning curve for content creators. Apostrophe has powered websites and web apps for organizations large and small for over a decade.
 
 #### Built With
 
 * [Node](https://nodejs.org/en/)
 * [Nunjucks](https://mozilla.github.io/nunjucks/)
 * [MongoDB](https://www.mongodb.com/)
 
-<!-- GETTING STARTED -->
 ## Getting Started
 
-To get started with ApostropheCMS, follow these steps to set up a local development environment.
+To get started with ApostropheCMS, follow these steps to set up a local development environment. For more detail, refer to the [Getting Started tutorial](https://docs.apostrophecms.org/getting-started/creating-your-first-project.html) in the documentation.
 
 #### Prerequisites
 
 We recommend installing the following with [Homebrew](https://brew.sh/) on macOS. If you're on Linux, you should use your package manager (apt or yum). If you're on Windows, we recommend the Windows Subsystem for Linux.
 
 | Software | Minimum Version | Notes
 | ------------- | ------------- | -----
-| Git  | Any
 | Xcode  | Current | Only needed on macOS
 | Node.js | 8.x | Or better
 | npm  | 6.x  | Or better
 | MongoDB  | 3.6  | Or better
 | Imagemagick (optional)  | Any | GIF support, faster image uploads
 
 
-#### New Project
-Our recomended way to start a new project with ApostropheCMS is to use the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli). Alternatively, you can simply clone this repo and `npm install`.
+#### Installation
+Our recomended way to start a new project with ApostropheCMS is to use the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli).
+
 ##### Install the Apostrophe CLI
  ```sh
   npm install -g apostrophe-cli
  ```
 ##### Create a project
- ```sh
- apos create-project <shortname-without-spaces>
- ```
-
-`<shortname-without-spaces>` represents the "shortname" of the application. It will be used by default as a MongoDB database name and a basis for cookie names, etc. You will find this as the shortName property in app.js.
-
+```sh
+apos create-project wonderful-project
+```
+
+##### Install dependencies
+```sh
+cd wonderful-project && npm install
```

Comment:
FWIW, the install and user creation step can be automated with `apos create-project wonderful-project --setup`. Skips two extra steps.